app/src/main/java/com/evercheck/flagly/developeroptions/FeatureFlagHandlerPresenter.kt

```diff
@@ -28,16 +31,24 @@ class FeatureFlagHandlerPresenter @Inject constructor(
 
     override fun onViewReady() {
         view?.setup()
-        setupFeatureFlagValues()
+    }
+
+    override fun onLoadFeatureFlagValues(search: String) {
+        this.search = search
+        this.setupFeatureFlagValues()
     }
```

Comment:
modify setupFeatureFlagValues() to receive an optional param, to avoid the use of global variables

Reply:
If necessary, since when updating the status, it consults all the information again and the filter would be lost